plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/bean/CoinmateBeanV1.java

```diff
@@ -161,6 +165,23 @@ private TransactionCluster createBuySellTransactionCluster() {
         return cluster;
     }
 
+    private TransactionCluster createRebateTransactionCluster() {
+        TransactionCluster cluster = new TransactionCluster(
+            new FeeRebateImportedTransactionBean(
+                id,             //uuid
```

Comment:
make these comments in one column (tabbed to same place), or just after code, but dont mix it

Reply:
you missed this comment